packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,31 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Use custom ScrollController when scrollController is set', (WidgetTester tester) async {
+        final ScrollController customController = ScrollController(initialScrollOffset: 10);
+        await tester.pumpWidget(
+          MaterialApp(
+            home: ReorderableListView(
+              scrollController: customController,
+              children: const <Widget>[
+                SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
+              ],
+              onReorder: (int oldIndex, int newIndex) { },
+            ),
+          )
+        );
+
+        final ReorderableListView listView = tester.widget(
+          find.byType(ReorderableListView),
+        );
+
+        listView.scrollController.jumpTo(20);
+
+        expect(customController.offset, 20);
```

Comment:
Instead of checking the controller's offset value here, check the actual value position of the box relative to the list view. For example,
```dart
        // Update scroll controller and see if list view updates accordingly.
        customController.animateTo(
          40.0,
          duration: const Duration(milliseconds: 200),
          curve: Curves.linear,
        );
        await tester.pumpAndSettle();

        listViewTopLeft = tester.getTopLeft(
          find.byType(ReorderableListView),
        );
        firstBoxTopLeft = tester.getTopLeft(
          find.byKey(firstBox),
        );
        expect(firstBoxTopLeft.dy, listViewTopLeft.dy - 40.0);
```